File: src/main/java/com/ftn/mbrs/service/VoziloService.java
package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Vozilo;

public interface VoziloService {

	public Vozilo save(Vozilo vozilo, Long modelVozilaId, Long karticaId);
	
	public Vozilo update(Vozilo vozilo);

	public String delete(Long id);
	
	public Vozilo findOne(Long id);
	
	public List<Vozilo> findAll();
}

File: src/main/resources/static/js/services/voziloService.js
angular.module('gremoNaElektrikoApp.VoziloService', [])
	.factory('VoziloService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/vozilo');
		}
		
		factory.createVozilo = function(vozilo , modelVozilaId, karticaId) {
			return $http.post('/gremoNaElektriko/vozilo'+'/'+modelVozilaId+'/'+karticaId, vozilo);
		}

		factory.deleteVozilo = function(id) {
			return $http.delete('/gremoNaElektriko/vozilo/'+id);
		}
		
		factory.updateVozilo = function(vozilo) {
			return $http.put('/gremoNaElektriko/vozilo', vozilo);
		}
		
		return factory;
	});